pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -971,6 +1018,201 @@ func (r *ReconcileClusterDeployment) reconcileInstallingClusterInstall(cd *hivev
 	return r.reconcileExistingInstallingClusterInstall(cd, ci, logger)
 }
 
+// ensureMetadataJSONSecret makes sure there is a Secret containing metadata.json, and sets/updates
+// clusterMetadata.MetadataJSONSecretRef to point to it. The caller is responsible for folding the
+// (potentially) modified clusterMetadata back into the cd and Update()ing it.
+// If forceUpdate is true, we will ensure the contents of the Secret's metadata.json key are equal
+// to metadataJSON. If forceUpdate is false, we will only update labels iff necessary.
+func (r *ReconcileClusterDeployment) ensureMetadataJSONSecret(
+	cd *hivev1.ClusterDeployment,
+	clusterMetadata *hivev1.ClusterMetadata,
+	metadataJSON []byte,
+	forceUpdate bool,
+	logger log.FieldLogger) error {
```

Comment:
```suggestion
	logger log.FieldLogger,
) error {
```

Cosmetic only suggestion

Reply:
Ack. (I generally like golang's opinionated whitespace rules, but I feel like they really missed the mark on this one. An extra indent for 2nd+ lines of long conditionals or function defs would fix it.)